python/rpdk/typescript/codegen.py

```diff
@@ -229,9 +230,14 @@ def _remove_build_artifacts(deps_path):
 
     @staticmethod
     def _make_build_command(base_path, build_command=None):
+        if sys.platform == "win32":
+            slash = "\\"
+        else:
+            slash = "/"
+
         command = (
             "npm install --optional "
-            + f"&& sam build --debug --build-dir {base_path}/build"
+            + f"&& sam build --debug --build-dir {base_path}{slash}build"
```

Comment:
should we just use `os.path.join` instead?

Reply:
Good point!